### PR TITLE
[HERA-256] [LMS] [Lesson Title] updates hera title xblock student view

### PR DESCRIPTION
**Pull request by Golub-Sergey**
**Description:** updates hera title xblock student view

**Youtrack:** [ticket](https://youtrack.raccoongang.com/issue/HERA-256)

### CODE REVIEW

File: hera_xblocks/hera_title/hera_title/hera_title.py [hunk 1]
```diff
@@ -13,22 +17,40 @@ class HeraTitleXBlock(StudioEditableXBlockMixin, XBlock):
     """
     editable_fields = ('data',)
 
-    display_name = String(default="Hera Title")
```
Comment (sendr) [inline hera_xblocks/hera_title/hera_title/hera_title.py]: please turn this field back

File: hera_xblocks/hera_title/hera_title/hera_title.py [hunk 2]
```diff
@@ -13,22 +17,40 @@ class HeraTitleXBlock(StudioEditableXBlockMixin, XBlock):
...
     data = JSONField(default={})
 
+    @property
+    def heading(self):
+        return self.data.get("heading",)
+
+    @property
+    def img_url(self):
+        return self.data.get("imgUrl", [])
```
Comment (sendr) [inline hera_xblocks/hera_title/hera_title/hera_title.py]: `imgUrl` - string with only one url to an image

File: hera_xblocks/hera_title/hera_title/hera_title.py [hunk 3]
```diff
@@ -13,22 +17,40 @@ class HeraTitleXBlock(StudioEditableXBlockMixin, XBlock):
     """
     editable_fields = ('data',)
 
-    display_name = String(default="Hera Title")
     data = JSONField(default={})
 
+    @property
+    def heading(self):
+        return self.data.get("heading",)
```
Comment (sendr) [inline hera_xblocks/hera_title/hera_title/hera_title.py]:
```suggestion
        return self.data.get("heading")
```

Comment (Golub-Sergey) [reply hera_xblocks/hera_title/hera_title/hera_title.py]: Done

File: hera_xblocks/hera_title/hera_title/hera_title.py (hunk 2 above)
Comment (Golub-Sergey) [reply hera_xblocks/hera_title/hera_title/hera_title.py]: Done

File: hera_xblocks/hera_title/hera_title/hera_title.py (hunk 1 above)
Comment (Golub-Sergey) [reply hera_xblocks/hera_title/hera_title/hera_title.py]: Done

**Pull request merged**